contracts/erc721/ERC721CommonEnumerable.sol

```diff
@@ -32,6 +32,16 @@ contract ERC721CommonEnumerable is ERC721Common, ERC721Enumerable {
         return ERC721Common.isApprovedForAll(owner, operator);
     }
 
+    /// @dev Calls ERC721Common.setApprovalForAll to manage pre-approvals
+    /// related to OpenSea's gas-free listing functionality.
+    function setApprovalForAll(address operator, bool approved)
+        public
+        virtual
+        override(ERC721Common, ERC721, IERC721)
+    {
+        ERC721Common.setApprovalForAll(operator, approved);
+    }
+
```

Comment:
Good question. To be honest, I added this to satisfy the CI pipeline. On my local machine this wasn't needed I I didn't fully understand why yet. Will investigate.

Reply:
This is because `ERC721Enumerable` inherits from `ERC721`. Therefore, both `ERC721Enumerable` and `ERC721Common` have a `setApprovalForAll` method and you need to specify which one to use here.